app/navigation/nonsipp/ReceiveTransferNavigator.scala

```diff
@@ -55,7 +55,7 @@ object ReceiveTransferNavigator extends JourneyNavigator {
 
     case page @ ReportAnotherTransferInPage(srn, index, secondaryIndex) =>
       if (userAnswers.get(page).contains(true)) {
-        refineV[OneTo50](secondaryIndex.value + 1) match {
+        refineV[OneTo5](secondaryIndex.value + 1) match {
```

Comment:
This is where the new method to fill in index gaps should come. Are you going to add that in a separate branch @narendrapaduchuri or you planned to do it in this branch?

Reply:
I will fix in my next branch @kberec